pom.xml

```diff
@@ -100,6 +100,7 @@
         <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
         <sonar.language>java</sonar.language>
         <sonar.verbose>true</sonar.verbose>
+        <jsr107.api.version>1.0.0</jsr107.api.version>
```

Comment:
I've removed it from jet-core and keep this one since it is needed for jet-all too